task_1/4.js

```diff
@@ -0,0 +1,12 @@
+/*
+Напишите функцию isEven(n), которая принимает один параметр, число, и
+возвращает четное ли оно.
+То есть функция может возвращать два значения: четное или нечетное.
+isEven(4); // true
+isEven(5); // false
+*/
+
+const isEven = (number) => number % 2 === 0 ? true : false;
```

Comment:
redundant ternary operators. `number % 2 === 0` already return boolean, so that no need to specify it